storage/create_bucket_upload_object/main.tf

```diff
@@ -0,0 +1,23 @@
+# Create new storage bucket in the US multi-region
+# with standard storage
+
+# [START storage_create_bucket_upload_object_tf]
+resource "google_storage_bucket" "static" {
+  project       = "<var>PROJECT_ID</var>"
```

Comment:
Please remove the `project` argument.

Reply:
I understand that `project` is typically included in the `provider` block. For the Terraform quickstart (https://cloud.google.com/storage/docs/terraform-create-bucket-upload-object), though, specifying `project` as part of the `resource` block is the simplest/quickest way to get users creating a Terraform file that works. Specifying `project` as part of the `provider` block would introduce another step in the quickstart.

Reply:
@cbonnie - During the CICD testing, this will issues. You can use data "google_project" instead.

Example
```
data "google_project" "project" {
    # project_id = "uncomment this like and update your project name here"
}
```